scripts/Utils/CurveData.reds

```diff
@@ -0,0 +1,60 @@
+
+public native struct CurvePointFloat {
+    public native let point: Float;
+    public native let value: Float;
+}
+
+public native struct CurveDataFloat {
+    public static native func GetSize(self: script_ref<CurveDataFloat>) -> Uint32;
+    public static native func GetPoint(self: script_ref<CurveDataFloat>, index: Uint32) -> CurvePointFloat;
+    public static native func SetPoint(self: script_ref<CurveDataFloat>, index: Uint32, point: CurvePointFloat) -> Void;
+    public static native func SetPointValue(self: script_ref<CurveDataFloat>, index: Uint32, point: Float, value: Float) -> Void;
```

Comment:
I'd add a counterpart method `GetPointValue(self: script_ref<CurveDataFloat>, index: Uint32, out point: Float, out value: Float)`. On C++ side, parameters will be `float* aPoint` and `T* aValue`.